cmd/utils.go

```diff
@@ -264,6 +264,62 @@ func getProjectDir(config *RootCommandConfig) (string, error) {
 	return config.ProjectDir, nil
 }
 
+func setProjectName(config *RootCommandConfig, projectName string) error {
+	dir, perr := getProjectDir(config)
```

Comment:
As mentioned previously, I would remove this call from here and do it in the caller. Pass in two strings - `projectDir` and `projectName`.

Reply:
Done